tabbycat/results/result.py

```diff
@@ -944,13 +946,18 @@ def teamscorebyadj_field_margin(self, adj, side):
     def teamscorebyadj_field_score(self, adj, side):
         return self.scoresheets[adj].get_total(side)
 
+    def _teamscore_score_component(self, adj, side):
+        if self.tournament.pref('teamscore_includes_ghosts'):
```

Comment:
Australs results seem flipped, is this line missing a `not`?

Reply:
Fixed. Prefer avoiding a "if not, else" construction, so swapped the returns.